recipes/annotated-types/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "annotated-types" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/annotated_types-{{ version }}.tar.gz
+  sha256: fae0ca44ad43b7e54dbb02b22137885b1cb1cb7a312fe9fe7a6d77a0e5b0443a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
```

Comment:
I like following NEP 29 as a default, if there's a CFEP or some other preferences I'm happy to drop this to 3.7 or (marginally less preferred) 3.6

Reply:
No preference here, we'll probably drop 3.7 before too long. I only went with `>=3.7` since it matched your original `skip` and it matches upstream.

Reply:
Ah, good catch, I'll stick with what the maintainers specified 👍